src/IceRpc.Templates/Templates/IceRpc-Protobuf-DI-Client/.template.config/template.json

```diff
@@ -0,0 +1,57 @@
+{
+    "$schema":"http://json.schemastore.org/template",
+    "author":"ZeroC",
+    "classifications":[
```

Comment:
is there a DI classification we can/should add here?

Reply:
I don't see anything in the default templates:
https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-new-sdk-templates